myapp/spec/spec_helper.rb

```diff
@@ -39,6 +39,14 @@
     mocks.verify_partial_doubles = true
   end
 
+  config.before(:each) do
+    Bullet.start_request if defined?(Bullet)
+  end
+
+  config.after(:each) do
+    Bullet.end_request if defined?(Bullet)
+  end
```

Comment:
I checked. bullet.perform_out_of_channel_notifications
is a configuration option in the Bullet gem to enable "out of channel" notifications in test environments. This means that Bullet will warn you when it detects an N+1 query or unused preload via logging, console output, or other configured notifications, rather than just via regular HTTP response headers or JavaScript warnings. I don't think it's necessary, but it would be more convenient to include it

Reply:
I see, thank you!
Please attach the URL that you used as a reference when you investigated. 

It depends on how you want to use it. 
I think that it is difficult to detect rspec via HTTP response header or JavaScript, so it is better to have console output or logging?

Reply:
I thought so too, so I added it at [.](https://github.com/Fablic/training/pull/1685/commits/fe9a5acf6c1cb36d7514535c72270abeaa215c11) Just realized it was automatically removed when running rubocop, I'll re-add it.